dnSpy/dnSpy/MainApp/AppCommandLineArgs.cs

```diff
@@ -262,5 +269,62 @@ public AppCommandLineArgs()
 		}
 
 		public IEnumerable<Tuple<string, string>> GetArguments() => userArgs.Select(a => Tuple.Create(a.Key, a.Value));
+
+
+
+		public void ShowHelp() {
```

Comment:
It should show the message in a dialog box. This is what most other UI programs do from what I've seen.